Gruntfile.js

```diff
@@ -1,18 +1,20 @@
 module.exports = function (grunt) {
     grunt.initConfig({
         pkg: grunt.file.readJSON('package.json'),
-        // Concatenate any type of files 
+        // Concatenate any type of files
         concat: {
             css: {
                 src: [
-                    'node_modules/bootstrap/dist/css/bootstrap.min.css'
+                    'node_modules/bootstrap/dist/css/bootstrap.min.css',
+                    'cssfiles/*'
```

Comment:
Also, trailing commas help to keep diffs smaller and more readable.

https://www.puppetcookbook.com/posts/trailing-commas.html